test/fixtures/master.cjs

```
@@ -0,0 +1,52 @@
+    NODE_ENV: 'dev',
+  },
+})
+  .on('fork', function(worker) {
+    console.warn('[%s] [test:worker:%d] new worker start', Date(), worker.process.pid);
+  })
+  .on('listening', function(worker, address) {
+    console.warn('[%s] [test:worker:%d] listening on %j', Date(), worker.process.pid, address.port);
+    process.send('listening');
+  })
+  .on('disconnect', function(worker) {
+    const propertyName = worker.hasOwnProperty('exitedAfterDisconnect') ? 'exitedAfterDisconnect' : 'suicide';
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)